src/ert/ensemble_evaluator/dispatch.py

```diff
@@ -10,51 +10,37 @@
 logger = logging.getLogger(__name__)
 
 
-class BatchingDispatcher:
-    def __init__(self, loop, timeout, max_batch=1000):
+class BatchingDispatcher:  # pylint: disable=too-many-instance-attributes
+    def __init__(self, timeout, max_batch=1000):
```

Comment:
I guess that the batch size remains on 1000 then?

Reply:
yepp that's right, there doesn't seem to be a need to modify that parameter at this time